fpl/fpl.py

```diff
@@ -39,11 +41,19 @@
                     position_converter, scale, team_converter)
 
 
-class FPL():
+class FPL:
     """The FPL class."""
 
     def __init__(self, session):
         self.session = session
+        static = requests.get(API_URLS["static"]).json()  # use synchronous request
```

Comment:
Is there a reason you are using `requests` to send the request instead of `aiohttp`? It feels a bit weird to me to have `requests` as a required package when it's only used once for this reason.